packages/contracts-bedrock/test/vendor/eas/eip1271/EIP1271Verifier.t.sol

```diff
@@ -0,0 +1,411 @@
+    bytes32 private constant ATTEST_TYPEHASH = 0xfeb2925a02bae3dae48d424a0437a2b6ac939aa9230ddc55a1a76f065d988076;
+
+    // =============================================================
+    //                          TEST STATE
+    // =============================================================
+    TestEIP1271Verifier public verifier;
+    MockEIP1271Signer public mockSigner;
+    address public recipient;
+    uint256 public signerPrivateKey;
+    address public signer;
+
+    // =============================================================
+    //                         ERROR TYPES
+    // =============================================================
+    error InvalidNonce();
+    
+    // =============================================================
+    //                           EVENTS
+    // =============================================================
+    event NonceIncreased(uint256 oldNonce, uint256 newNonce);
+
+    // =============================================================
+    //                           SETUP
+    // =============================================================
+    /// @dev Deploys contracts and initializes test state
+    function setUp() public {
+        verifier = new TestEIP1271Verifier("EAS");
+        mockSigner = new MockEIP1271Signer();
+        recipient = makeAddr("recipient");
+        signerPrivateKey = 0xA11CE;
+        signer = vm.addr(signerPrivateKey);
+    }
+    // =============================================================
+    //                    INTERNAL HELPERS
+    // =============================================================
+    /// @dev Helper function to hash typed data for EIP712
+    function _hashTypedDataV4(DelegatedAttestationRequest memory request) internal view returns (bytes32) {
+        bytes32 structHash = keccak256(
+            abi.encode(
+                ATTEST_TYPEHASH,
+                request.attester,
+                request.schema,
+                request.data.recipient,
+                request.data.expirationTime,
+                request.data.revocable,
+                request.data.refUID,
+                keccak256(request.data.data),
+                request.data.value,
+                verifier.getNonce(request.attester),
+                request.deadline
+            )
+        );
+
+        return keccak256(
+            abi.encodePacked(
+                "\x19\x01",
+                verifier.getDomainSeparator(),
+                structHash
+            )
+        );
+    }
+    // =============================================================
+    //                      BASIC STATE TESTS
+    // =============================================================
+    /// @dev Tests initial verifier configuration
+    function testInitialState() public view {
+        assertEq(verifier.getName(), "EAS");
+        assertEq(verifier.getNonce(signer), 0);
+    }
+
+    // =============================================================
```

Comment:
These comments are nice, its generally better to match the style of the codebase than diverge even if the divergence is better